get_version.py

```diff
@@ -23,4 +23,4 @@
 ])
 
 # Set the output filename to the name of the board and the version
-env.Replace(PROGNAME=f'mobiflight_{env["PIOENV"]}_{firmware_version.replace(".", "_")}')
+env.Replace(PROGNAME=f'{env["PIOENV"]}_{firmware_version.replace(".", "_")}')
```

Comment:
As now firmware names should start with the supporter/user name, `mobiflight` is moved to the environments